openapi/social/social.yaml

```
@@ -142,6 +142,86 @@ paths:
           in: query
           name: dryRun
           description: 'If `true`, will simply return any errors this Message will get from posting. Will return an empty 200 if there are none.'
+    get:
+      summary: List Social Messages
+      operationId: get-messages
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/messages'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+                        format: uri
+                      first:
+                        type: string
+                        format: uri
+                        description: Provides a link back to the first page of results.
+                      next:
+                        type: string
+                        format: uri
+                        description: The URI at which the next page of results can be gotten from.
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
+        
+         An endpoint dedicated to listing all the posts made by the business filtered by the selected parameters.
+
+      parameters:
+        - schema:
+            type: string
+            example: AG-12345
+          in: query
+          name: 'filter[businessId]'
```

We should make the name match the other API operations. 

```suggestion
          name: 'filter[businessLocation.id]'
```